src/components/postView/postComponents/__tests__/PostCard.test.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { shallow } from "enzyme";
+import PostCard from "../PostCard";
+
+// test component rendering
+describe("<PostCard /> rendering", () => {
+    const wrapper = shallow(<PostCard />);
+    // test if PostCard component renders
+    it("should be able to render <PostCard />", () => {
+        wrapper;
+    });
+});
```

Comment:
Good, but maybe add some tests to check that important child components have rendered